src/services/dumper/agent-nodejs-dumper.ts

```diff
@@ -0,0 +1,118 @@
+import type { ConfigInterface } from '../../interfaces/project-create-interface';
+
+import toValidPackageName from '../../utils/to-valid-package-name';
+import AbstractDumper from './abstract-dumper';
+
+export default class AgentNodeJsDumper extends AbstractDumper {
+  private env: any;
+
+  private readonly DEFAULT_PORT = 3310;
+
+  private readonly isLinuxOs: any;
+
+  constructor(context) {
+    const { assertPresent, env, isLinuxOs } = context;
+
+    assertPresent({
+      env,
+      isLinuxOs,
+    });
+
+    super(context);
+
+    this.env = env;
+    this.isLinuxOs = isLinuxOs;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  get templateFolder() {
+    return 'agent-nodejs/';
+  }
+
+  writePackageJson(dbDialect: string, applicationName: string) {
+    const dependencies: any = {
```

Comment:
mongoose dependency is missing here, coming soon alongside mongoose models

Reply:
Not needed: https://docs.forestadmin.com/developer-guide-agents-nodejs/data-sources/provided-data-sources/mongoose